### DISCUSSION TITLE
Plex Media Server LXC edit Preferences.xml

### DISCUSSION THREAD

**Discussion by Hoosier-IT** [tteck/Proxmox#2086] [Help Needed]
from console, how can I edit the Preferences.xml?
![Capture](https://github.com/tteck/Proxmox/assets/8570961/f700fd2d-cc90-4343-b34a-452b3ba1a042)

**Comment by tteck** [comment]
https://support.plex.tv/articles/204281528-why-am-i-locked-out-of-server-settings-and-how-do-i-get-in/

**Reply by Hoosier-IT** [reply]
Thanks. Was able to finally get to the file.  However, the noted attribute/key pairs are not listed.

Linux
Quit/exit your Plex Media Server, so it’s not actively running
In your text editor, remove the following attribute/key pairs from the Preferences.xml file:
PlexOnlineHome="1"
PlexOnlineMail="jane@example.com"
PlexOnlineToken="RanD0MHex1DecIm4LtoKeNheR3"
PlexOnlineUsername="ExampleUser"
Save the edited file
Launch your Plex Media Server

**Reply by tteck** [reply]
What happens when going to `IP:32400/web`?

**Reply by Hoosier-IT** [reply]
https://github.com/tteck/Proxmox/assets/8570961/4b3fcd31-97e8-4970-bace-a8c2cf19b1d0

**Reply by tteck** [reply]
I see

**Reply by Hoosier-IT** [reply]
Best just to delete the LXC and try again? Assuming it must run privileged?

**Reply by tteck** [reply]
Or restore a backup

**Reply by steRnbear87** [reply]
I was able to figure this out using SSH Tunnel to the plex lxc.
I used the tutorial from this link:

https://www.plexopedia.com/plex-media-server/windows/putty-ssh-tunnel-windows/

**Comment by Hoosier-IT** [comment]
Neither reinstalling or the SSH Tunnel seems to resolve the issue for me.

**Reply by Hoosier-IT** [reply]
Found the issue and sorted, using your link and changing the host name to the ip in Putty. @steRnbear87 thanks!

**Comment by pear235** [comment]
Hey @Hoosier-IT, mind telling how did you solve it step by step? I don't have them lines mentioned in the plex webpage article in Preferences.xml

**Comment by Hoosier-IT** [comment]
https://github.com/tteck/Proxmox/discussions/2086#discussioncomment-7748087

Use the IP assigned
